code/modules/antagonists/cult/cult_items.dm

```diff
@@ -395,7 +395,7 @@
 	icon_state = "cult_sharpener"
 	uses = 1
 	increment = 5
-	max = 40
+	max = 30
```

Comment:
bruh longswords START at 30

Reply:
Yeah that's the point, it doesn't work with the whetstones. It's already a free e sword, it's silly to make it a free desword.

Reply:
the main draw of the desword is the projectile reflection, not the damage